tempesta_fw/t/functional/long_body/test_long_request.py

```diff
@@ -0,0 +1,68 @@
+""" Testing for long body in request """
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2017 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+import unittest
+import body_generator
+import os
+
+from testers import stress
+from helpers import tf_cfg, control, tempesta, remote, wrk
+
+class RequestTestBase(stress.StressTest):
+    """ Test long request """
+    config = "cache 0;\n"
+    root = "/tmp/long_body/"
```

Comment:
There is variable `client.workdir` in `tf_cfg` which points to `/tmp/client` by default. Use it as root dir for your directories and files.

Reply:
fixed